3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -88,8 +88,8 @@ Di seguito sono elencati i requisiti minimi necessari per l'esecuzione dell'appl
     columns: 3,
     [*Componente*], [*Versione*], [*Riferimenti*],
 
-    [Docker],[$>=$ 24.0.7],[https://docs.docker.com/],
-    [Docker-compose],[$>=$ 2.23.3],[https://docs.docker.com/compose/],
+    [Docker],[$>=$ 24.0.7],[https://docs.Docker.com/],
+    [Docker-compose],[$>=$ 2.23.3],[https://docs.Docker.com/compose/],
```

Comment:
Ripropongo modifica in queste due righe: "docker" con la "d" minuscola nei due link